src/PhpParser/Node/BetterNodeFinder.php

```diff
@@ -576,8 +576,6 @@ public function resolvePreviousNode(Node $node): ?Node
     /**
      * @api
      *
-     * @deprecated Use StmtsAwareInterface instead
-     *
```

Comment:
@TomasVotruba `resolveNextNode()` and `resolvePreviousNode()` are new public method, so can't be deprecated, and there is no solution for next/prev of Expr, Identifier, Name yet, except lookup of `$file->getOldtokens()` hack. 

see PR: 

- https://github.com/rectorphp/rector-src/pull/3896 

for the reasoning :). This is verify current start/end token inside the Stmt, if it is on edge, then jump to next stmt_key + 1/-1 for next/prev statement.

 I reverted the deprecated note, except, if needed, use better comment instead.